renderer/src/hooks/StationRewards.tsx

```diff
@@ -1,6 +1,7 @@
 import { useEffect, useMemo, useState } from 'react'
 import { getScheduledRewards } from 'src/lib/station-config'
 import useWallet from 'src/hooks/StationWallet'
+import { formattedFilToBigInt } from 'src/lib/utils'
```

Comment:
eventually we should refactor the system never to pass formatted FIL around, only bigint or strings